tests/travis/mysql.travis.xml

```diff
@@ -12,7 +12,6 @@
         <var name="tmpdb_host" value="localhost" />
         <var name="tmpdb_username" value="travis" />
         <var name="tmpdb_password" value="" />
-        <var name="tmpdb_name" value="doctrine_tests_tmp" />
```

Comment:
totally fine IMO as MySQL allows connecting without dbname. The temp connection is only for database creation AFAIK. Also travis is green :)